controllers/rest/operation_initializers.go

```diff
@@ -678,6 +625,23 @@ func RouteInitializer(ctxt context.Context, api *RESTAPI, path string, method st
 	if entityFactory == nil {
 
 	}
+
+	if controller == nil {
+		//once no controller is set, the default controller and middleware is added to the path
+		controller, err = api.GetController("DefaultResponseController")
+		if err != nil {
+			api.e.Logger.Warnf("unexpected error initializing controller: %s", err)
+			return ctxt, fmt.Errorf("controller '%s' set on path '%s' not found", "DefaultResponseController", path)
+		}
+
+		middlewares := GetOperationMiddlewares(ctxt)
+		//there are middlewareNames let's add them
+		if middleware, _ := api.GetMiddleware("DefaultResponseMiddleware"); middleware != nil {
```

Comment:
So this being a middleware might be a good thing (I was initially thinking it wasn't). You can in theory pair this middleware with the other standard middleware and have the "Default" controller attached in all instances (and stop using the other standard controllers)